payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/TextFieldUI.kt

```diff
@@ -147,7 +147,7 @@ fun TextField(
                 this.contentDescription = contentDescription
                 this.editableText = AnnotatedString("")
             },
-        enabled = enabled,
+        enabled = enabled && textFieldController.enabled,
```

Comment:
It feels weird to me that there is an enabled field passed in and the textFieldController also specifies enabled? 

The TextFieldUI enabled parameter changes as a result of changes between elements in the form, whereas the textFieldController.enabled is more the textField itself independently wanting to be displayed a certain way.

I think a comment could help here, or maybe in the place where enabled is passed into the TextFieldUI the textFieldController is checked.

Reply:
Added a javadoc comment to the `enabled` parameter.